src/generators/static/README.md.hbs

```diff
@@ -55,9 +55,11 @@ You can authenticate with Azure Active Directory using the [Azure Identity libra
 npm install @azure/identity
 ```
 
-You will also need to register a new AAD application and grant access to {{ serviceName}} by assigning the suitable role to your service principal (note: roles such as `"Owner"` will not grant the necessary permissions).
+You will also need to **register a new AAD application and grant access to {{ serviceName}}** by assigning the suitable role to your service principal (note: roles such as `"Owner"` will not grant the necessary permissions).
 Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables: `AZURE_CLIENT_ID`, `AZURE_TENANT_ID`, `AZURE_CLIENT_SECRET`.
 
+For more information about how to create an Azure AD Application check out [this guide](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal).
```

Comment:
Remove the locale i.e. `en-us` from the url here to allow people at different locations to fall back to their own locale

Reply:
Good catch, I opened PR https://github.com/Azure/autorest.typescript/pull/1155 for this.